### PR TITLE
Typings for to-time

### PR DESCRIPTION
**Pull request by hafuta**
Please fill in this template.
- [v] Prefer to make your PR against the `types-2.0` branch.
- [v] The package does not provide its own types, and you can not add them.
- [v] Test the change in your own code.
- [v] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [v] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [v] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [v] Run `tsc` without errors.
- [v] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header.

### CODE REVIEW

**Comment by hafuta** [comment]
Passed Travis CI test

File: to-time/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        /**
+         * Initialize a new TimeFrame object with years 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromYears(years: number) : TimeFrame;
+    }
+
+}
+
+declare module 'to-time' {
```
Comment (vvakame) [inline to-time/index.d.ts]: could you stripe `declare module 'to-time' {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred

File: to-time/to-time-tests.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/// <reference path="index.d.ts" />
```
Comment (vvakame) [inline to-time/to-time-tests.ts]: we can remove this line.

File: to-time/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```
Comment (vvakame) [inline to-time/tsconfig.json]: could you try to use `"strictNullChecks": true,`?

**Comment by paulvanbrenk** [comment]
This PR seems to be abandoned, please re-open when you made the requested changes.

**Pull request closed without merging**